cli/embedded-app/babel.config.js

```diff
@@ -1,4 +1,15 @@
 module.exports = {
-  presets: ['@vue/app'],
-  exclude: ['../../dist/*.js']
+  presets: [
+    '@vue/app',
+    [
+      '@babel/preset-env',
+      {
+        useBuiltIns: 'entry',
+        targets: {
+          ie: '11'
```

Comment:
Its probably not a huge deal.  But this could in theory remove conversions that firefox/chrome need but IE doesn't. 
 It's more accurate to include all of your targets.

Reply:
maybe just use the new defaults query 7.5 added? https://babeljs.io/blog/2019/07/03/7.5.0#defaults-browserlist-query-in-babel-preset-env-8897-https-githubcom-babel-babel-pull-8897

Reply:
Ah yeah, that was me rushing / being lazy. Thanks for catching me on it.